kubewarden/kubewarden-controller
# fix: add validation for empty rules objects

## Description

* Add validation for empty rules objects in the rules array.
* Ensure that each rules object contains one operation and at least one of API groups, API versions, or resources.


Fixes #361 




## Test


* Create a ClusterAdmissionPolicy/AdmissionPolicy with an empty Rules object:
    ```yaml
    apiVersion: policies.kubewarden.io/v1
    kind: ClusterAdmissionPolicy
    metadata:
      name: test-bug
    spec:
      mode: protect
      module: ghcr.io/kubewarden/policies/disallow-service-nodeport:v0.1.2
      mutating: false
      policyServer: default
      rules:
      - {}
      settings: null
    ```
* Observe that this request is denied.
